packages/arb-token-bridge-ui/src/util/RetryableUtils.ts

```diff
@@ -62,8 +62,8 @@ export const getRetryableTicketExpiration = async ({
     isLoadingError = false,
     isExpired = false
 
-  let daysUntilExpired = 0,
-    expirationDate = 0
+  let daysUntilExpired = 0
+  const expirationDate = 0
```

Comment:
i'm unsure about this one
faced this when i tried to fix it on the first eslint PR

if you look at the `expirationDate` in the `try` block and the `return` block, it seems to make more sense if this is `let` and the one in the try block is an assignment without initialisation

need @dewanshparashar's input

Reply:
Good point, we should have a eslint rule for shadowing to avoid those cases IMO.

Reply:
That's true, doesn't make sense. Initializing with a `const` here should ideally not override in the try block - that creates a scope of it's own. Outside of the try block it will still remain 0. 
We should change it to - 
<img width="665" alt="image" src="https://user-images.githubusercontent.com/7558499/205046486-798dcb02-5ab1-41d2-b2a6-b40664dd9013.png">

Reply:
Most probably the previous code didn't work as the `expirationDate` returned, was the one defined outside of the try/catch block.